core/src/services/memcached/backend.rs

```diff
@@ -82,6 +86,30 @@ impl MemcachedBuilder {
         self.config.default_ttl = Some(ttl);
         self
     }
+
+    /// Set the tls connect on.
+    pub fn enable_tls(mut self, enable_tls: bool) -> Self {
+        self.config.enable_tls = enable_tls;
+        self
+    }
+
+    /// Set the tls cafile path.
+    pub fn cafile(mut self, cafile: &str) -> Self {
```

Comment:
We don't need to support specify `cafile`. Users that need this can just use `SSL_CERT_DIR`.